src/views/ErrorPage.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="pa-6 text-center">
+    <h1 class="text-sm-large">
+      DON'T PANIC.
+    </h1>
+    <p v-if="!isDevMode">
+      Something went wrong 😧
+      <br />
+      Please email
+      <a href="mailto:tech@extinctionrebellion.nl">
+        <!-- eslint-disable-next-line vue/html-closing-bracket-newline -->
+        tech@extinctionrebellion.nl</a
+      >
+      for help.
+    </p>
+    <p v-else>
+      Failed to connect to the server. Please make sure you have
+      <a href="https://github.com/xrnl/aginun#install">
+        <!-- eslint-disable-next-line vue/html-closing-bracket-newline -->
+        set the server API key</a
+      >
+      .
+      <br />
+      If the problem persists please contact
+      <a href="mailto:tech@extinctionrebellion.nl">
+        tech@extinctionrebellion.nl
+      </a>
+    </p>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "ErrorPage",
+  data: () => ({
+    isDevMode: process.env.NODE_ENV === "production",
```

Comment:
small typo here :wink: 
```suggestion
    isDevMode: process.env.NODE_ENV !== "production",
```

Reply:
ahah thanks!